autometa/validation/gdown_fileIDs.csv

```diff
@@ -0,0 +1,168 @@
+community,dataset,file,file_id
```

Comment:
We should be able to remove this from the PR now that it is hosted on google drive. Correct?

Reply:
Yep!